packages/react/test/specs/behaviors/treeItemBehavior-test.tsx

```diff
@@ -0,0 +1,49 @@
+import { treeItemBehavior } from 'src/lib/accessibility'
```

Comment:
suggest to introduce e2e tests for Tree's _focus_ behavior - will be glad to assist if necessary.

Reply:
already created task for that, thanks!